docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,34 @@
+# Content-cache with Openstack/Swift storage
+
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and reducing the number of requests to the [swift](https://docs.openstack.org/swift/latest/) server.
+
+This guide will demonstrate how to deploy this charm along with OpenStack/Swift storage.
+
+First, connect to your openstack instance with your credentials, sourcing an .rc file:
+```
+source myfile.rc
+```
+if you don't have any credentials check [here](https://docs.openstack.org/zh_CN/user-guide/common/cli-set-environment-variables-using-openstack-rc.html) for more information
+Then list all the objects on the openstack containers in debug mode:
+```
+openstack object list default -vv
```

Comment:
This is listing the "default" container which may or may not exist. We should mention that the user should change this (in the command below if we collapse this into just offering one command) to the container they want to serve content from.